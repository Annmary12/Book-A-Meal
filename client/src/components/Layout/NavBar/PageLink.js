import React from 'react';
import classNames from 'classnames';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';
import styles from './navbar.scss';

const PageLink = ({ pathname, to, text }) => (
  <Link
    className={classNames({
      [styles.active]: pathname === to
    })}
    to={to}
  >
    {text}
  </Link>
);

PageLink.propTypes = {
  pathname: PropTypes.string.isRequired,
  to: PropTypes.string.isRequired,
  text: PropTypes.string.isRequired
};

export default PageLink;
