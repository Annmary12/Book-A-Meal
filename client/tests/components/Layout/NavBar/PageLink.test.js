import React from 'react';
import PageLink from '../../../../src/components/Layout/NavBar/PageLink';

describe('PageLink component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<PageLink to="test" />);

    expect(wrapper).toMatchSnapshot();
  })
})