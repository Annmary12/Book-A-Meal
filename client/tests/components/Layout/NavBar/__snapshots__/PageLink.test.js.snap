// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLink component should match snapshot 1`] = `
<Link
  className=""
  replace={false}
  to="test"
/>
`;
